### DISCUSSION TITLE
Hammerspoon with Stream Deck Mk2

### DISCUSSION THREAD

**Discussion by spritelite** [Hammerspoon/hammerspoon#3116] [General]
Hello dear contributors,

is there already a planning for the next release of Hammerspoon with support of the Stream Deck Mk2? I had tried to compile the changes myself with Xcode, but was unfortunately not very successful due to the many error messages.

**Comment by cmsj** [comment]
I can't promise a timeline, but yes, the next release will support the Mk2. We have everything we need to make it work, it's just a matter of getting the bits all squared away (or rather, having the time to do that).

**Reply by latenitefilms** [reply]
I'm pretty insanely busy this week, but when I catch my breathe, I'll connect up all my Stream Deck's and test out the pull request.

**Comment by spritelite** [comment]
Thank you for your comments. Is there an estimate on the time horizon when to expect the new version?

**Reply by cmsj** [reply]
By incredible coincidence, I released 0.9.94 about a minute ago :)

**Reply by spritelite** [reply]
Perfect. Good news!

**Reply by spritelite** [reply]
And it is working. Thank you and have a nice weekend.
